controllers/monitoring_alert.go

```diff
@@ -180,7 +213,15 @@ func (r *ManagedMCGReconciler) reconcileMonitoringResources() error {
 	}
 	for i := range promRuleList.Items {
 		obj := promRuleList.Items[i]
-		utils.AddLabel(obj, monLabelKey, monLabelValue)
+		if obj.GetName() == "noobaa-prometheus-rules" {
```

Comment:
It seems that this is just a "cleaning" task to revert a label set in previous versions. If i am not wrong, maybe a brief comment explaining "why" we do this will make the code more easy to understand.